rsmtool/modeler.py

```diff
@@ -452,9 +448,9 @@ def train_lasso_fixed_lambda_then_lr(self, df_train, feature_columns):
             The SKLL learner object
         fit : statsmodels.RegressionResults
             A statsmodels regression results object.
```

Comment:
The treatment of "statsmodels" I will also neglect from now on.

```suggestion
            A ``statsmodels`` regression results object.
```